### PR TITLE
infoschema: fix panic in 'SELECT * FROM information_schema.processlist'

### PR DESCRIPTION
**Pull request by tiancaiamao**
### What problem does this PR solve? 

Fix https://github.com/pingcap/tidb/issues/9836

### What is changed and how it works?

This bug is introduced in https://github.com/pingcap/tidb/pull/9681, where the type of `ProcessInfo.Command` is changed from `string` to `byte`, but the corresponding code in the infoschema package is not updated.

### Check List 

Tests 

 - Unit test

### CODE REVIEW

**Comment by tiancaiamao** [comment]
PTAL @jackysp @xiekeyi98

File: infoschema/tables.go [hunk 1]
```diff
@@ -649,7 +649,7 @@ func dataForProcesslist(ctx sessionctx.Context) [][]types.Datum {
 			pi.User,
 			pi.Host,
 			pi.DB,
-			pi.Command,
+			mysql.Command2Str[pi.Command],
```
Comment (ngaut) [inline infoschema/tables.go]: There are multiple place that build process information, could you abstraction a function ?

Comment (tiancaiamao) [reply infoschema/tables.go]: Done

**Pull request merged**